### DISCUSSION TITLE
Feedback about changes in sliders reporting back

### DISCUSSION THREAD

**Discussion by frank1119** [jpcima/ysfx#56] [General]
FYI: With the plugin from Jan 12th 2022, changes from sliders in code did not properly report back to the host, even when slider_automate() was called. The plugin from May 28th 2022 solves this problem also. 

It might have been solved earlier, but I did not check all other versions in between. 

This is just to confirm that this works.

**Comment by jpcima** [comment]
Good to know, thanks.
Yes unfortunately it's some of these threading issues which are very host-specific.
I don't test a vast number of hosts, so these reports are very welcome.
